src/i18n/i18n.csv

```diff
@@ -22,7 +22,7 @@ General (i18n),all,All,Tous,Alle,Të gjithë,جميع,Բոլորը,Всички,
 ,srch-menus,Search and menus,Recherche et menus,Soek en spyskaarte,Kërko dhe menutë,البحث والقوائم,Որոնել եւ Ընտրացանկերը,Търсене и менюта,搜索和菜單,搜索和菜单,Vyhledávání a nabídky,Zoeken en menu's,Otsi ja menüüd,Suche und Menüs,Αναζήτηση και μενού,खोज और मेनू,Keresés és menük,Leit og valmyndir,Cari dari menu,Ricerca e menu,ᕿᓂᕈᑎ ᐊᒻᒪ ᓱᔾᔪᑏᑦ,検索とメニュー,검색 및 메뉴,Meklēšanas un ēdienkartes,Ieškoti ir meniu,Wyszukaj i menu,Pesquisa e menus,Pesquisa e menus,Поиск и меню,Vyhľadávanie a ponuky,Búsqueda y menús,ค้นหาและเมนู,Arama ve menüler,Пошук і меню,Tìm kiếm và thực đơn
 ,menu,Menu,Menu,Menu,Menu,قائمة الطعام,Մենյու,Меню,選單,菜单,Menu,Menu,Menüü,Menü,Μενού,मेनू,Menü,Valmynd,Menu,Menu,ᓱᔾᔪᑎᑦ,メニュー,메뉴,Izvēlne,Meniu,Menu,Menu,Menu,Меню,Menu,Menú,เมนู,Menü,Меню,Thực đơn
 ,menu-close,Close menu,Fermer le menu,Maak die menu,Mbylle menu,إغلاق القائمة,Փակել մենյուն,Затворете менюто,關閉菜單,关闭菜单,Zavřete menu,Sluit het menu,Sulge menüü,Schließen Sie das Menü,Κλείστε το μενού,मेनू बंद,Zárja be a menüt,Loka valmynd,Tutup menu,Chiudere il menu,ᓱᔾᔪᑎᑦ ᒪᑐᓕ,メニューを閉じる,메뉴를 닫습니다,Aizveriet izvēlni,Uždaryti meniu,Zamknij menu,Fechar o menu,Fechar o menu,Закройте меню,Zavrite menu,Cierre el menú,ปิดเมนู,Menüyü kapatmak,Закрийте меню,Đóng menu
-,overlay-close,Close overlay,Fermer la fenêtre superposée,Maak die oortrek,Mbylle mbulesë,إغلاق تراكب,Փակեք կափարիչ,Затворете наслагването,關閉疊加,关闭叠加,Zavřete se šablonou,Sluit het dialoogvenster,Sulge pop-up,Schließen Sie das Pop-up,Κλείστε το αναδυόμενο παράθυρο,पॉपअप बंद,Zárja be az overlay,Loka yfirlagið,Tutup hamparan,Chiudere la sovrapposizione,ᖃᓕᕇᑦ ᒪᑐᓕ,オーバーレイを閉じる,오버레이에게 닫습니다,Aizveriet pārklāt,Uždaryti perdangos,Zamknij nakładkę,Feche a sobreposição,Feche a sobreposição,Закройте вспомогательных линий,Zavrite so šablónou,Cierre la superposición,ปิดการแสดงข้อมูล,Bindirme kapatmak,Закрийте допоміжних ліній,Đóng lớp phủ
+,overlay,overlay,la fenêtre superposée,Maak die oortrek,Mbylle mbulesë,إغلاق تراكب,Փակեք կափարիչ,Затворете наслагването,關閉疊加,关闭叠加,Zavřete se šablonou,Sluit het dialoogvenster,Sulge pop-up,Schließen Sie das Pop-up,Κλείστε το αναδυόμενο παράθυρο,पॉपअप बंद,Zárja be az overlay,Loka yfirlagið,Tutup hamparan,Chiudere la sovrapposizione,ᖃᓕᕇᑦ ᒪᑐᓕ,オーバーレイを閉じる,오버레이에게 닫습니다,Aizveriet pārklāt,Uždaryti perdangos,Zamknij nakładkę,Feche a sobreposição,Feche a sobreposição,Закройте вспомогательных линий,Zavrite so šablónou,Cierre la superposición,ปิดการแสดงข้อมูล,Bindirme kapatmak,Закрийте допоміжних ліній,Đóng lớp phủ
```

Comment:
There is no issue with the content change. 

However, let keep the same i18n key

```suggestion
,overlay-close,overlay,la fenêtre superposée,Maak die oortrek,Mbylle mbulesë,إغلاق تراكب,Փակեք կափարիչ,Затворете наслагването,關閉疊加,关闭叠加,Zavřete se šablonou,Sluit het dialoogvenster,Sulge pop-up,Schließen Sie das Pop-up,Κλείστε το αναδυόμενο παράθυρο,पॉपअप बंद,Zárja be az overlay,Loka yfirlagið,Tutup hamparan,Chiudere la sovrapposizione,ᖃᓕᕇᑦ ᒪᑐᓕ,オーバーレイを閉じる,오버레이에게 닫습니다,Aizveriet pārklāt,Uždaryti perdangos,Zamknij nakładkę,Feche a sobreposição,Feche a sobreposição,Закройте вспомогательных линий,Zavrite so šablónou,Cierre la superposición,ปิดการแสดงข้อมูล,Bindirme kapatmak,Закрийте допоміжних ліній,Đóng lớp phủ
```

Note: I didn't reviewed the other translation.